ParkingLot/src/main/java/services/ParkingSpotManager/FourWheelerParkingSpotManager.java

```diff
@@ -1,11 +0,0 @@
-package services.ParkingSpotManager;
-
-import models.ParkingSpots.ParkingSpot;
-
-import java.util.List;
-
-public class FourWheelerParkingSpotManager extends ParkingSpotManager{
-    public FourWheelerParkingSpotManager(List<ParkingSpot> parkingSpotList) {
```

Comment:
Is this class having any other separate functionality which it wants to override from parkingspotmanager?

Reply:
As of now no. But we might need to override the findNearestSpot method.